nixos/tests/sway.nix

```diff
@@ -66,30 +80,28 @@ import ./make-test-python.nix ({ pkgs, lib, ...} :
     machine.wait_for_file("/run/user/1000/wayland-1")
     machine.wait_for_file("/tmp/sway-ipc.sock")
 
-    # Test XWayland:
+    # Test XWayland (foot does not support X):
     machine.succeed(
         "su - alice -c 'swaymsg exec WINIT_UNIX_BACKEND=x11 WAYLAND_DISPLAY=invalid alacritty'"
     )
     machine.wait_for_text("alice@machine")
     machine.send_chars("test-x11\n")
     machine.wait_for_file("/tmp/test-x11-exit-ok")
-    print(machine.succeed("cat /tmp/test-x11.out"))
-    machine.screenshot("alacritty_glinfo")
+    machine.copy_from_vm("/tmp/test-x11.out")
```

Comment:
The `copy_from_vm` is very cool, thanks!
I'm just wondering if we should keep the `print`. My main reason for that was that it's available via the build logs as well (ofborg and Hydra).

Reply:
We could have both if we want